### DISCUSSION TITLE
Country not found while trying 2FA by SMS

### DISCUSSION THREAD

**Discussion by kiranbishwo** [community/community#106573] [New to GitHub]
i have tried 2FA in my account , i am from nepal but i cant find nepal on the country select field.
![Screenshot 2024-02-07 112518](https://github.com/community/community/assets/69898491/1b5857a1-a7fc-4054-9164-651225fa666b)

**Comment by Anixton** [comment]
Looks like your country is not supported for SMS auth (https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication)

You can use GitHub Mobile or Security keys as your 2FA .

GitHub Mobile: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-github-mobile
Security keys: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-security-key
TOTP: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app

Please verify this answer, have a great day!

**Comment by AndreaGriffiths11** [comment]
@Anixton is correct, thank you for answering. I'm afraid, we can currently only offer SMS authentication in countries where our SMS providers can guarantee a high deliverability rate. Unfortunately, their service is not reliable enough in Nepal for us to offer SMS authentication as an option, at this time.
